modio_repo/downloader/mod_files.py

```diff
@@ -4,10 +4,13 @@
 import modio
 from modio.client import Mod as ApiMod
 from modio.enums import TargetPlatform
+from modio.entities import ModFilePlatform
 
 from modio_repo.models import Mod, PcModFile, QuestModFile
 from modio_repo.utils import log
 
+def contains_targetplatforms(modplatforms: list[ModFilePlatform], targetplatforms: list[TargetPlatform]):
```

Comment:
not a fan of this function, it would probably be better to just use a set() overlap - whats there originally was a quick fix to get it working after api changes.

Reply:
you can remove this change. it was a temporary fix on my end since the files wasnt downloading on my end.

Reply:
i think you're actually onto something there, it seems that this is the root cause of the issues with it not updating. i'll merge as is and probably rewrite it later.